javascript/GoogleMapField.js

```diff
@@ -49,21 +50,33 @@
 				$('.cms-edit-form').addClass('changed');
 			}
 		}
-
+		
 		function updateZoom() {
 			zoomField.val(map.getZoom());
 		}
+		
+		function updateBounds() {
+			var bounds = JSON.stringify(map.getBounds().toJSON());
+			boundsField.val(bounds);
+		}
+
+		function zoomChanged() {
+			updateZoom();
+			updateBounds();
+		}
 
 		function centreOnMarker() {
 			var center = marker.getPosition();
 			map.panTo(center);
 			updateField(center);
+			updateBounds();
```

Comment:
Actually mapClicked doesn't need to update bounds. Deleted updateBounds() in mapClicked. So I think updateBounds should be separate from "updateLatLng". What do you think?